client/src/App.tsx

```diff
@@ -8,25 +8,29 @@ import "ace-builds/src-noconflict/mode-python";
 import "ace-builds/src-noconflict/theme-monokai";
 
 const apiUrl = "https://ufgjji253b.execute-api.us-east-1.amazonaws.com/prod";
-const defaultJsonObject = '{\n\t"foo": 5 \n}';
+const defaultJsonObject = '{\n\t"foo": 5 \n"barBaz": "hello"}';
```

Comment:
```suggestion
const defaultJsonObject = '{\n\t"foo": 5, \n\t"barBaz": "hello"}';
```

Reply:
oops, nice catch. Weird that I didn't see it when developing locally